src/resmoke/resmoke_suite.rs

```diff
@@ -0,0 +1,369 @@
+//! Representation of a resmoke suite file.
```

Comment:
This code was also moved and split out from the old resmoke code, but it was changed a lot. I switched from using a generic yaml loader for the resmoke configs to using serde. It cleaned up the code a lot. Instead of having super deeply nested queries about the yaml structure, we just define `struct`s the model the structure and work with them directly.

Reply:
Nice, this looks a lot cleaner